src/rules/__tests__/no-interpolation-inline-snapshot.test.ts

```diff
@@ -0,0 +1,40 @@
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+import resolveFrom from 'resolve-from';
+import rule from '../no-interpolation-inline-snapshot';
+
+const ruleTester = new TSESLint.RuleTester({
+  parser: resolveFrom(require.resolve('eslint'), 'espree'),
+  parserOptions: {
+    ecmaVersion: 2017,
+  },
+});
+
+ruleTester.run('no-interpolation-inline-snapshot', rule, {
+  valid: [
```

Comment:
Could you add some more tests for cases like:

* when expect is called without a matcher or modifier
* when expect is called with a dangling modifier (i.e no matcher)
* when a property is accessed on `expect` (i.e `expect.toHaveAssertions()`
* when a method called `toMatchInlineSnapshot` is called with interpolation, both by itself and on an object that looks suspiciously similar to `expect`

Reply:
added in https://github.com/jest-community/eslint-plugin-jest/pull/553/commits/b6c50ec9a1ae5cdea27c0e6af9002e0c421d15b7

Reply:
Awesome, thanks!

If you could check one more in that has two or more interpolations, just for completeness, that would be cool but not blocking.

Reply:
added in 96eaf9b